roles/inject_scorecard_metadata/tasks/main.yml

```diff
@@ -48,14 +48,17 @@
   environment:
     PATH: "{{ testing_bin_path}}:{{ lookup('env', 'PATH') }}"
 
-- name: "Load CSV into a csv_vars"
-  include_vars:
-    file: "{{ csv_path }}"
-    name: csv_vars
+- name: "Read csv.yaml file"
+  shell: "cat {{ csv_path }}"
+  register: csv_data
+
+- name: "Set facts for csv_data"
+  set_fact:
+    csv_vars: "{{ csv_data.stdout | from_yaml }}"
 
 - name: "Get spec from csv_vars"
   set_fact:
-    csv_vars_spec: "{{ csv_vars['spec'] | default({}) }}"
+    csv_vars_spec: "{{ (csv_vars | from_yaml)['spec'] | default({}) }}"
```

Comment:
Do we have to convert the csv_vars again here when it is already done on line#57 ?

Reply:
If we used from_yaml only in the initial part of declaring csv_vars and no further, nested variables of that fact(csv_vars) would be unaccessible by ansible.